### ISSUE TITLE
module01_feedback

### ISSUE THREAD

**Issue by gogleva** [tavareshugo/slcu_r_course#11]
24 participants (18 giving feedback)!
- explain paths better (slower?) -- find a video to send
- more times about R in general (45 minutes)
- no 'here' in the exercises!
- explain what should be done
- compiled markdown - to show
- not install tidyverse?/use graphical interface
- do not use Rmarkdown --> extract info and put it in the normal markdown
- check that packages are installed properly --> add more instructions
- use spare laptop with tidyverse installed/ not troubleshoot in place
- mooore time for R!
- explain variable and functions
- do R introduction for next module (02)
- need more volunteers to help
-

**Comment by tavareshugo** [comment]
A few more detailed comments and suggestions from Katie:

- Work from a normal R-script rather than Rmd
    - Start with empty script
    - Then "write-along-with-me" kind of delivery of materials, so people have a chance to
write and keep up with the material
    - working from Rmd is hard to follow which code chunk is evaluated by the instructor
- Still have an Rmd with the course materials, which forms the website page - people can look at this after the course if they want to see an example of R Markdown
- Also show an example of a "real life" example of why R Markdown can be useful, for example, recently I shared this code with someone we've been in touch for Katie's project: https://github.com/tavareshugo/20171005_seed_models. In that case it was useful to use Markdown because I could mix text and code.
- Do not spend much time (or any time at all) on exercises. Session is short, so breaking with exercises is perhaps not needed. Instead, solve small problems "together" as a group. For example: "If I wanted to do a violin plot instead, how do you think we would have to change this code?". Then have a couple of people answer and then show it. Also, "homework" does the job of the exercises - at home people can try to consolidate the materials.

**Comment by tavareshugo** [comment]
To compensate for unclear session on R, could try and email a normal script with the few lines of code needed to read a CSV file.

Something like:

```
# Script to read data into R
# Any line starting with the # (hash symbol) is a comment - R "ignores" these lines

# First load all the libraries needed for this script
library(tidyverse)

# Check what the working directory is at the moment
#  this is what "folder" R is working from at the moment
getwd()

# Set working directory to the course materials folder
# Change this to be suitable to your situation
setwd("/home/hugo/Desktop/slcu_r_course/module01_data_and_files")

# Read the CSV file and store the table in an object called "my_surveys"
## In this case, the CSV file is located in the "course_data" folder
my_surveys <- read_csv("../course_data/dataset_tidy.csv")

# View the loaded table
View(my_surveys)
```

**Comment by tavareshugo** [comment]
Email participant mentioning the following:

- We have updated the materials based on people's feedback. In the "Reading Data into R" section there is now a small intro to R and also a solution of the reading data exercise. Please note that we have moved the data for the lesson into the "module01_data_and_files" folder to make it less confusing. It is best if you download the course folder again. 
- Can come at 1pm if you want some extra help
- Please check the software installation section again to ensure you have all the necessary packages installed: https://tavareshugo.github.io/slcu_r_course/#software_installation
